stafeev_artem/task_02.cpp

```diff
@@ -18,17 +18,13 @@ class Graph {
   }
 
  private:
-  EdgeId get_new_edge_id() { return k_edges_count_++; }
+  EdgeId get_new_edge_id() { return edges_count_++; }
 
-  VertexId get_new_vertex_id() { return k_vertices_count_++; }
+  VertexId get_new_vertex_id() { return vertices_count_++; }
 
   bool has_vertex(VertexId id) const {
-    if (std::any_of(vertices_.begin(), vertices_.end(),
-                    [&id](Vertex vert) { return vert.id() == id; })) {
-      return true;
-    }
-
-    return false;
+    return std::any_of(vertices_.begin(), vertices_.end(),
+                       [id](Vertex vert) { return vert.id() == id; });
```

Comment:
1) Не примитивные данные лучше передавать по конст ссылке
2) Переименуй переменную и старайся избегать сокращений без крайней необходимости
```suggestion
                       [id](const Vertex& vertex) { return vertex.id() == id; });
```